### ISSUE TITLE
No binaries for arm (32-bit v7)

### ISSUE THREAD

**Issue by BretFisher** [replicatedhq/ship#1150]
In order to support Raspberry Pi's, can you add a Linux build target of `arm` for go binary releases?

**Comment by laverya** [comment]
Are you actually using ship from a Raspberry Pi?

**Comment by BretFisher** [comment]
Yes. I teach Kubernetes in online courses and a small, but increasing group of them are using 32-bit and 64-bit Linux on arm/v7 and arm/v8.  I've also got a surprising number of Darwin arm64 users (Apple M1 arm/v8) starting to show up (so that's a 2nd build target request I guess).  I'll also be putting this request in kots :)

**Comment by laverya** [comment]
I can definitely support this for kots!
